frontend/src/APIClients/BaseAPIClient.ts

```diff
@@ -73,9 +73,13 @@ const accessTokenInjectionLink = new ApolloLink(
 
 const refreshDirectionalLink = new RetryLink().split(
   (operation) =>
-    ["Refresh", "ResetPassword", "Login", "Signup_Register"].includes(
-      operation.operationName,
-    ),
+    [
+      "Refresh",
+      "ResetPassword",
+      "Login",
+      "Signup_Register",
+      "Courses",
```

Comment:
Can you double check that the module editor still works with a stale account?

Reply:
<img width="1714" alt="image" src="https://user-images.githubusercontent.com/36215359/188534061-a4d94d44-b815-4698-b47c-b7586fb92009.png">
nope